zebrad/Cargo.toml

```
@@ -3,15 +3,15 @@
 name = "zebrad"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
 license = "MIT OR Apache-2.0"
-version = "1.0.0-rc.8"
+version = "1.0.0-rc.9"
 repository = "https://github.com/ZcashFoundation/zebra"
 
 # Settings that impact compilation
 edition = "2021"
 
 # Zebra is only supported on the latest stable Rust version. Some earlier versions might work.
 # Zebra's code uses features introduced in Rust 1.68, or any later stable release.
-rust-version = "1.68"
+rust-version = "1.70"
```

I just found this out: this setting will make `cargo` 1.68 and 1.69 exit with an error, so it might not be what we want:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field

If you've run `cargo check` with 1.68 then I think it's fine letting people use it.

```suggestion
rust-version = "1.68"
```